### DISCUSSION TITLE
Exclude offline sites from `exp:channel:entries` output

### DISCUSSION THREAD

**Discussion by robsonsobral** [ExpressionEngine/ExpressionEngine#2895] [Feature Requests]
Offline sites are not excluded from queries:

```
{exp:channel:entries channel="service" site="not 0" category="{cat_ids}"
disable="custom_fields|categories|category_fields|member_data|pagination"
status="not closed" limit="5"}
```

Isn't possible to prepare a new website while there are others online on MSM.

**Comment by robsonsobral** [comment]
This should be related to the use of the `channel` parameter both on **Channel Module** as also on **Search Module**.

**Comment by intoeetive** [comment]
Just noting that implementation would need to be done with a separate parameter. 

Some setups might have offline sites and still use those as data sources for other sites

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4484
